web/styles/modal.css

```diff
@@ -215,21 +235,21 @@
 
         .modal__content {
             /* When showing read receipts, we use simplebar
-            to make the list scrollable.  It requires this to
-            be flex. */
+             * to make the list scrollable.  It requires this to
+             * be flex. */
             display: flex;
 
             /* Setting a minimum height prevents the loading indicator
-               appearing/disappearing from resizing the modal in the
-               common case that one is requesting read receipts for
-               direct messages. */
+             * appearing/disappearing from resizing the modal in the
+             * common case that one is requesting read receipts for
+             * direct messages. */
```

Comment:
When contributing to a codebase, you need to configure your editor to not restyle every file you edit.